Doc/c-api/list.rst

```diff
@@ -51,6 +51,17 @@ List Objects
    ``len(list)`` on a list object.
 
 
+.. c:function:: int PyList_Clear(PyObject *list)
+
+   Remove all items from *list*. Similar to:
+   ``PyList_SetSlice(L, 0, PY_SSIZE_T_MAX, NULL)``.
```

Comment:
```suggestion
   ``PyList_SetSlice(list, 0, PY_SSIZE_T_MAX, NULL)``.
```
What does it mean to be similar? Does it have exactly the same effect? If not, what is the difference

Reply:
`PyList_Clear(list)` is the same as `PyList_SetSlice(list, 0, PY_SSIZE_T_MAX, NULL)`. Internally, both end up calling `list_clear()`: same code path.